### PR TITLE
Page title and design improve 

### PR DESCRIPTION
**Pull request by almog8k**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                       |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,9 +2,9 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="icon" type="image/svg+xml" href="/configIcon.svg" />
```
Comment (CptSchnitz) [inline index.html:5]:
```suggestion
    <link rel="icon" type="image/svg+xml" href="/config-icon.svg" />
```

File: src/App.tsx [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
 import { SnackbarProvider } from 'notistack';
 import { queryClient } from './api/tanstack/queryClient';
 
 function App() {
   return (
     <>
-      <ThemeProvider theme={materialUITheme}>
+      <ThemeProvider>
         <QueryClientProvider client={queryClient}>
           <SnackbarProvider>
```
Comment (CptSchnitz) [inline src/App.tsx:13]: check that snackbar changes to dark mode

File: src/components/pageTitle/pageTitle.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+export const PageTitle: React.FC<PageTitleProps> = ({ title }) => {
+  const location = useLocation();
+
+  useEffect(() => {
+    document.title = title;
+  }, [location, title]);
+
+  return null;
+};
```
Comment (CptSchnitz) [inline src/components/pageTitle/pageTitle.tsx:16]:
```tsx
          <>
            <PageTitle title="My Home Page" />
            <Home />
          </>
```

File: src/pages/configs/configTable/configTable.tsx [hunk 4]
```diff
@@ -2,7 +2,7 @@ import { Link, useNavigate } from 'react-router-dom';
 import { GenericTable, TableColumn } from '../../../components/genericTable/genericTable';
 import { ClipboardCopyButton } from '../../../components/clipboardCopyButton/clipboardCopyButton';
 import { ActionMenu } from '../../../components/actionMenu/actionMenu';
-import { Box, MenuItem, Tooltip, Typography } from '@mui/material';
+import { Box, Link as MuLink, MenuItem, Tooltip, Typography } from '@mui/material';
```
Comment (CptSchnitz) [inline src/pages/configs/configTable/configTable.tsx:5]:
```suggestion
import { Box, Link as MuiLink, MenuItem, Tooltip, Typography } from '@mui/material';
```

**Review by CptSchnitz — APPROVED** [review]

File: src/App.tsx (hunk 2 above)
Comment (almog8k) [reply src/App.tsx:13]: it does not

File: src/pages/configs/configTable/configTable.tsx (hunk 4 above)
Comment (almog8k) [reply src/pages/configs/configTable/configTable.tsx:5]: done

**Pull request merged**